src/components/modals/BtcSendModal.vue

```diff
@@ -344,9 +346,11 @@ export default defineComponent({
             amount.value = maxSendableAmount.value;
         }
 
-        function updateFee(newFeePerByte: number) {
+        interface FeeSelectorEvent {fee: number; durationConfirmation: number}
+        function updateFee({ fee: newFeePerByte, durationConfirmation: _duration }: FeeSelectorEvent) {
             const isSendingMax = amount.value === maxSendableAmount.value;
             feePerByte.value = newFeePerByte;
+            durationConfirmation.value = _duration;
```

Comment:
I'm not a fan of using underscores for shadowed variables. In this case, you can use the same mechanism as the other variable, and call it `newDuration`.